node/src/chain_specs/calamari_pc.rs

```diff
@@ -177,126 +177,15 @@ fn calamari_dev_genesis(
 	}
 }
 
-pub fn calamari_testnet_config(id: ParaId) -> CalamariChainSpec {
-	let properties = calamari_properties();
-
-	// (controller_account, aura_id)
-	let initial_authorities: Vec<(AccountId, AuraId)> = vec![
-		(
-			// account id: dmvSXhJWeJEKTZT8CCUieJDaNjNFC4ZFqfUm4Lx1z7J7oFzBf
-			hex!["4294b2a716cea91dd008d694d264feeaf9f0baf9c0b8cbe3e107515947ed440d"].into(),
-			hex!["10814b2b41bf39155ef7b38bb2431056894ba71acc35cf0101c999fd69f9c357"]
-				.unchecked_into(),
-		),
-		(
-			// account id: dmxvZaMQir24EPxvFiCzkhDZaiScPB7ZWpHXUv5x8uct2A3du
-			hex!["b06e5d852078f64ab74af9b31add10e36d0438b847bc925fbacbf1e14963e379"].into(),
-			hex!["f2ac4141fee9f9ba42e830f39f00f316e45d280db1464a9148702ab7c4fcde52"]
-				.unchecked_into(),
-		),
-		(
-			// account id: dmud2BmjLyMtbAX2FaVTUtvmutoCKvR3GbARLc4crzGvVMCwu
-			hex!["1e58d3c3900c7ce6c6d82152becb45bf7bd3453fb2d267e5f72ca51285bca173"].into(),
-			hex!["f6284f9446db8f895c6cf02d0d6de6e67885a1e55c880ccac640ff4bc076df68"]
-				.unchecked_into(),
-		),
-		(
-			// account id: dmx4vuA3PnQmraqJqeJaKRydUjP1AW4wMVTPLQWgZSpDyQUrp
-			hex!["8a93e0f756448030dcb3018d25d75c7bf97a2e2ff15d02fd1f55bf3f2104fb5b"].into(),
-			hex!["741101a186479f4f28aa40fc78f02d7307ed3574e829aed76fdede5876e46a43"]
-				.unchecked_into(),
-		),
-		(
-			// account id: dmtwRyEeNyRW3KApnTxjHahWCjN5b9gDjdvxpizHt6E9zYkXj
-			hex!["0027131c176c0d19a2a5cc475ecc657f936085912b846839319249e700f37e79"].into(),
-			hex!["8ebf03bda1702d719f428bc0a4c7cfca010c44a48ef79752490818c901548d20"]
-				.unchecked_into(),
-		),
-	];
-
-	// root account: dmyBqgFxMPZs1wKz8vFjv7nD4RBu4HeYhZTsGxSDU1wXQV15R
-	let root_key: AccountId =
-		hex!["bc153ffd4c96de7496df009c6f4ecde6f95bf67b60e0c1025a7552d0b6926e04"].into();
-
-	CalamariChainSpec::from_genesis(
-		// Name
-		"Calamari Parachain Testnet",
-		// ID
-		"calamari_testnet",
-		ChainType::Local,
-		move || calamari_testnet_genesis(initial_authorities.clone(), root_key.clone(), id),
-		vec![],
-		Some(
-			sc_telemetry::TelemetryEndpoints::new(vec![(STAGING_TELEMETRY_URL.to_string(), 0)])
-				.expect("Calamari testnet telemetry url is valid; qed"),
-		),
-		Some(CALAMARI_PROTOCOL_ID),
-		Some(properties),
-		Extensions {
-			relay_chain: KUSAMA_RELAYCHAIN_DEV_NET.into(),
-			para_id: id.into(),
-		},
+pub fn calamari_testnet_config() -> Result<CalamariChainSpec, String> {
+	CalamariChainSpec::from_json_bytes(
+		&include_bytes!("../../../genesis/calamari-testnet-genesis.json")[..],
 	)
 }
 
-fn calamari_testnet_genesis(
```

Comment:
Remove this test gensis config, there're lots of complains about unused imports.